can_tools/scrapers/official/IL/il_demographics.py

```diff
@@ -64,9 +64,17 @@ def get(self) -> pd.DataFrame:
                     "race",
                 ).assign(county=name)
             )
+            parts.append(
```

Comment:
@sglyon -- Do I need to review these files? They don't seem to be actively running in airflow.

Reply:
Let's not review them here. I think I wrote them, but didn't activate them b/c we weren't decided on how to handle demographics. Leaving them as is is great